Automattic/woocommerce-payments
# Update to not open in new tab if onboarding not complete

Fixes #8812 

#### Changes proposed in this Pull Request









In some cases, we were opening the Stripe onboarding in a new tab, which can be confusing for merchants.

#### Testing instructions





* Using JN, spin up a site on this branch.
* Complete MOX and go to the first step of Stripe onboarding
* Click the link to go back to WooPayments. You should be taken to the `Payments > Overview` page.
* Click the `Finish Setup` task. (see screenshot).
* You should be redirected to the Stripe KYC in the same tab. Clicking the link to go back to WooPayments should again take you back to the Overview page.
* Scroll down to the `Account Tools` and click the `Finish setup` prompt there. Confirm this also opens in the same tab, and going back to WooPayments takes you back correctly.
* Use the Finish setup task to finish the onboarding and confirm you can successfully onboard an account.

![image](https://github.com/Automattic/woocommerce-payments/assets/11770181/e94b99b2-f8ec-47b5-bd5a-7a10a76ed179)


cc @timmy5685 


-------------------


**Post merge**
